docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:lts-buster-slim
+FROM node:fermium-buster-slim
```

Comment:
Not using hardcoded LTS is fine but fermium is the codename of node v14 whereas in `package.json` `engines` you say `^16.0.0` which would be the codename Gallium, see: https://nodejs.org/en/about/releases/

So we should use it here too - if that works.

To your question: we should definitely commit this as this defines which node version we use in the docker image and this should be the same as locally.
There is another Docker image which we should align: `docker/Dockerfile.production`
This should be using the same base image I think.

Reply:
Thanks for this info! I wasn't 100% sure what we should set this to for it to work for everyone. I will test out some updated options.

Reply:
Good call though, gallium is working for me. I've updated both Dockerfile locations.